src/lib.rs

```diff
@@ -28,6 +28,7 @@ use bdk::{
     Balance as BdkBalance, BlockTime, Error as BdkError, FeeRate, KeychainKind, SignOptions,
     SyncOptions as BdkSyncOptions, Wallet as BdkWallet,
 };
+use rand::{thread_rng, Rng};
```

Comment:
Would it be okay to use `thread_rng`, `Rng` this way?
```
use bdk::bitcoin::secp256k1::rand::{thread_rng, Rng};
```
This way we don't have to add `rand` to `Cargo.toml`

Reply:
Oh good idea. @notmandatory do I understand correctly that if this is coming from re-exports it's good to go? I think in this case `bdk` re-exports `bitcoin` which re-exports `secp256k1` which re-exports `rand`... lol.

Reply:
The latest version of `bdk` (0.24) updated `rand` version to 0.8, but now that I take a look it's not exported, so I think the best approach is to bring it in as our own dependency as you're doing.